stack-8.2.1.yaml

```diff
@@ -25,6 +25,7 @@ extra-deps:
 - hsimport-0.8.6
 - lsp-test-0.5.0.2
 - monad-dijkstra-0.1.1.2
+- mtl-2.2.2
```

Comment:
I think this can go
```suggestion
```

Reply:
Compare 
https://circleci.com/gh/haskell/haskell-ide-engine/4566?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link 
and 
https://circleci.com/gh/haskell/haskell-ide-engine/4583?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

I think it is neccessary, judging by the error message. Unfortunately, I am having troubles reproducing the problem due to [#4539](https://github.com/commercialhaskell/stack/issues/4539)

Reply:
Ah sorry, I didn’t see that. That seems reasonable